gobblin-audit/src/main/java/gobblin/audit/auditor/ValueAuditRuntimeMetadata.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.audit.auditor;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import org.apache.avro.Schema;
+import org.apache.commons.lang3.StringUtils;
+
+
+/**
+ * A container for table specific runtime Metadata required for auditing a table.
+ * Use {@link ValueAuditRuntimeMetadataBuilder} to instantiate a new {@link ValueAuditRuntimeMetadata}.
+ * <code>database, table and tableSchema</code> are required fields in the {@link ValueAuditRuntimeMetadata}.
+ * All other fields <code>phase, cluster, extractId, snapshotId, deltaId, avroFileName</code> are marked as {@value #MISSING_FIELD} if not present.
+ */
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+@ToString
+@EqualsAndHashCode
+public class ValueAuditRuntimeMetadata {
+
+  private static final String MISSING_FIELD = "NA";
+
+  private TableMetadata tableMetadata;
```

Comment:
please add javadocs to the fields. As of Lombok 1.12, those should be added to the javadocs of the generated setters/getters.